pom.xml

```diff
@@ -130,7 +120,7 @@
 		</repository>
 		<repository>
 			<id>org.geppetto-mvn-repo</id>
-            <url>https://raw.github.com/openworm/org.geppetto.maven/mvn-repo</url>
+            <url>https://raw.github.com/openworm/org.geppetto.maven/mvn-repo-nml1.5.2</url>
```

Comment:
@pgleeson is 1.5.2 available on mvn-repo? Is it the correct version? If yes should this change?